src/bld/package.js

```diff
@@ -43,6 +44,9 @@ const packager = async (srcDir, nwDir, outDir, platform, zip, releaseInfo) => {
   log.debug("Convert package.json buffer into JSON");
   let pkg = JSON.parse(buffer);
 
+  log.debug(`Rename NW executable to ${pkg.name}`);
+  await rename(`${outDir}/${getPlatformSpecificName(platform)}`, `${outDir}/${pkg.name}`);
```

Comment:
Isn't this going to incorrectly remove the `.exe` extension for Windows and `.app` extension for macOS?

Reply:
Should be fixed now

Reply:
I don't see any changes made to fix this? Still renaming to `${outDir}/${pkg.name}`.

Reply:
It should rename to `${outDir}/${pkg.name}` for Linux, `${outDir}/${pkg.name}.app` for Mac and `${outDir}/${pkg.name}.exe` for Windows

Reply:
My apologies. Not sure what I was looking at last night, but I see the correct code now.